other-builtin-functions/05_exploding_arrays/05_exploding_arrays.md

```diff
@@ -0,0 +1,73 @@
+# 04 Expanding arrays into new rows
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.3%2B-lightgrey)
```

Comment:
Flink 1.3?

Reply:
I've checked all the documentation since then and at least the docs say it was supported :)